waddrmgr/db.go

```diff
@@ -730,6 +745,8 @@ func fetchAddress(tx walletdb.Tx, addressID []byte) (interface{}, error) {
 	if err != nil {
 		return nil, err
 	}
+	used := fetchAddressUsed(tx, addrHash)
```

Comment:
I don't think we need the extra local here:

`row.used = fetchAddressUsed(tx, addrHash)`